src/playlist/PlaylistAny.hxx

```diff
@@ -34,7 +35,7 @@ class Storage;
  * music or playlist directory.
  */
 std::unique_ptr<SongEnumerator>
-playlist_open_any(const char *uri,
+playlist_open_any(const struct LocatedUri &located_uri,
```

Comment:
Don't use `struct` here - it's not necessary in C++.

Reply:
Fixed here and in other places